### PR TITLE
Add more tests for default-deny egress

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description
A few sentences describing the overall goals of the pull request's commits.
Please include
- the type of fix - (e.g. bug fix, new feature, documentation)
- some details on _why_ this PR should be merged
- the details of the testing you've done on it (both manual and automated)
- which components are affected by this PR
- links to issues that this PR addresses

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,7 @@ test-containerized: vendor run-etcd run-kubernetes-master
     -e LOCAL_USER_ID=$(LOCAL_USER_ID) \
     -v $(CURDIR)/.go-pkg-cache:/go/pkg/:rw \
     -v $(CURDIR):/go/src/github.com/$(PACKAGE_NAME):rw \
-    $(CALICO_BUILD) sh -c 'cd /go/src/github.com/$(PACKAGE_NAME) && make WHAT=$(WHAT) SKIP=$(SKIP) ut'
+    $(CALICO_BUILD) sh -c 'cd /go/src/github.com/$(PACKAGE_NAME) && make WHAT=$(WHAT) SKIP=$(SKIP) FOCUS=$(FOCUS) ut'
```
Comment (caseydavenport) [inline Makefile]: I need to revert this change.

File: lib/backend/k8s/k8s_fv_test.go [hunk 2]
```diff
@@ -153,6 +153,32 @@ var updateTypeStr = map[api.UpdateType]string{
 	api.UpdateTypeKVUpdated: "Updated",
 }
 
+func (c cb) GetSyncerUpdate(key string) *api.Update {
```
Comment (caseydavenport) [inline lib/backend/k8s/k8s_fv_test.go]: This is useful for getting an exact update that was received over the syncer (vs the other existing functions which just verify that an update was recieved).

File: run-uts [hunk 3]
```diff
@@ -25,7 +25,8 @@ test ! -z "$go_dirs"
 # Run tests in random order find tests recursively (-r).
 echo WHAT: $WHAT
 echo SKIP: $SKIP
-ginkgo -cover -coverpkg=${go_dirs} -r --skipPackage $SKIP $WHAT
+echo FOCUS: $FOCUS
+ginkgo -focus $FOCUS -cover -coverpkg=${go_dirs} -r --skipPackage $SKIP
```
Comment (caseydavenport) [inline run-uts]: Need to revert this as well.

File: Makefile (hunk 1 above)
Comment (bcreane) [reply Makefile]: ?
